### ISSUE TITLE
Search tabs are unintuitive

### ISSUE THREAD

**Issue by AxelBoldt** [mastodon/mastodon-ios#656]
When tapping the search icon in portrait mode, the tabs "Posts", "Hashtags", "News", "Community", "For You" appear.  It seems that the first three are not used and always empty(?) while "Community" shows the federated timeline and "For You" shows some relevant users. Once a search term is entered, the tabs change to "All", "People", "Hashtags", "Posts", presumably to display search results.

It is unintuitive to hide the federated timeline behind a search tab, preceded by three unused search tabs. It is also unintuitive to change the search tabs after entering a term.

**Comment by aarononeal** [comment]
Agreed. It's also constrained to a small column on the right instead of taking over the main content view, which gets difficult fast if you want to interact with any results.

**Comment by object01** [comment]
I'm not sure "Community" is the federated timeline? I only ever see posts from users on my instance.

**Comment by quinncomendant** [comment]
The tab names are somewhat ambiguous. The first time I opened the “Posts” tab, there was a welcome text at top, but it didn't explicitly describe where the posts come from ([screenshot](https://send.strangecode.com/f/PzQr2Hj.jpeg)). If each tab had some welcome text like this which actually described how the tab worked, it would help a lot. (Oh, actually I see that the web app does have a descriptive text at the top for each of its #Explore tabs, but the iOS app only showed me text in the posts tab.

I think one of the biggest design mistake is how different the structure of the iOS app is from the web app. For example, the names of these tabs are completely different from the tabs on the web app (Posts = Federated, Community = Local, right?). The iOS app's Search function should be named #Explore?

**Comment by westongpt** [comment]
I think Posts are _trending_ federated posts, not the same as the federated tab on web. But I agree with OP it's super confusing.

**Comment by alakoring** [comment]
I am looking for a way to search my own timeline (like you can in Twitteriffic, for example). This seems to be lacking, though maybe I am just confused by the terms.
